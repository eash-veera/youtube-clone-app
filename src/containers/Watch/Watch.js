import React,{ Component } from 'react';

export class Watch extends Component{
    render(){
        return (
            <div>Watch Component goes here !</div>
        );
    }
};