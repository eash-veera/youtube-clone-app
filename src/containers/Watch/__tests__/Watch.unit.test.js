import React from 'react';
import { shallow } from 'enzyme';
import { Watch } from '../Watch';

test('renderes <Watch /> with empty props', () =>{
    const wrapper = shallow(
        <Watch />
    );
    expect(wrapper).toMatchSnapshot();
});
