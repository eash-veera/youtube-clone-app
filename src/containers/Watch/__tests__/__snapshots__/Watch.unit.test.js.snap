// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renderes <Watch /> with empty props 1`] = `
<div>
  Watch Component goes here !
</div>
`;
